hnn_core/extracellular.py

```diff
@@ -28,6 +28,30 @@
 from .externals.mne import _validate_type, _check_option
 
 
+def calculate_csd2d(data, delta=1):
+    """current source density (csd) estimation
+
+    The three-point finite-difference approximation of the 
+    second spatial derivative for computing 1-dimensional CSD
+    Parameters
+    ----------
+    data : channels x times array
```

Comment:
```suggestion
    data : array, shape (n_channels, n_times)
```

Try to be as consistent as possible with the rest of the codebase